examples/Example_om2_forcing.yaml

```
@@ -0,0 +1,75 @@
+# Experiment Configuration File
+
+
+model_type: access-om2 # Specify the model ("access-om2", "access-om3", "access-esm1.5", or "access-esm1.6")
+repository_url: git@github.com:ACCESS-NRI/access-om2-configs.git
+start_point: "6b315e4" # Control commit hash for new branches
+test_path: prototype-0.2.0-om2-forcing # All control and perturbation experiment repositories will be created here; can be relative, absolute or ~ (user-defined)
+repository_directory: MC_25km_jra_ryf # Local directory name for the central repository (user-defined)
+
+control_branch_name: ctrl
+
+Control_Experiment:
+
+
+# Specific for perturbation experiments, if only one entry is provided for a key, it will be broadcast to all perturbations.
+# eg in below example, 
+# value:
+#   - INPUT/uas_..._Tropical_Pacific_Perturbation.nc
+# INPUT/uas_..._Tropical_Pacific_Perturbation.nc will be used for both perturbations, i.e., perturb1 and perturb2.
+#
+# another example
+# dimension:
+#   - ["spatial"]
+#   - ["temporal"]
+# where dimension is a list of lists. 
+# Each inner list corresponds to one perturbation experiment (column-wise).
+# Here a single inner list (eg ["spatial","temporal"]),
+# it will be broadcast to all perturbations, so each perturbation gets the plain list.
+Perturbation_Experiment:
+    Parameter_block1:
+        branches:
+            - perturb_1
+            - perturb_2
+        atmosphere/forcing.json:
+            uas:
+                cname:
+                    - uwnd_ai
+                    - uwnd_ai2
+                filename:
+                    - "INPUT/uas_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_{{year}}01010000-{{year}}12312100.nc"
+                    - "INPUT/uas_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_{{year}}01010000-{{year}}12312100.nc2"
+                perturbations:
+                - type:
+                    - scaling
+                    - offset
+                  dimension:
+                    - spatial
+                  value:
+                    - INPUT/uas_..._Tropical_Pacific_Perturbation.nc
+                  calendar:
+                    - forcing
+                  comment:
+                    - Time-independent increase in tropical Pacific zonal winds
+                - type:
+                    - offset
+                  dimension:
+                    - constant
+                  value:
+                    - 5
+                  calendar:
+                    - forcing
+                  comment:
+                    - ""
+                - type:
+                    - separable
+                  dimension:
+                    - ["spatial"]
+                    - ["temporal"]
```

```suggestion
                    - ["spatial", "temporal"]
```
need 2-element array for type "separable"